### ISSUE TITLE
Question Type

### ISSUE THREAD

**Issue by wbprice** [Code4HR/okcandidate-platform#181]
I'm working on setting up `SurveyCard` to respond to different question types.  The current model for `QuestionType` looks like:
```
{
    name: "Multiple Choice Only"
}
```

I want to formalize what "Multiple Choice Only" means to the UI without hardcoding the specific string "Multiple Choice Only" in the component code. Does it make sense enable/disable inputs for the survey card like so in the given `QuestionType`?

```
{
    name: "Multiple Choice Only",
    multipleChoice: true,
    sentiment: false
}
```

**Comment by ttavenner** [comment]
Agreed.

**Comment by wbprice** [comment]
`QuestionType` and `SurveyCard` behavior

- When both `multipleChoice` and `sentiment` is true, display both radio buttons and heart range selector.  **Heart range selector describes range of intensity of opinion.**
- when `multipleChoice` is true and `sentiment` is false, display only radio buttons.
- when `multipleChoice` is false and `sentiment` is true, display only heart range selector. **Heart range selector describes range of disagreement to agreement**

**Comment by wbprice** [comment]
Fixed in #182
